src/fonduer/parser/parser.py

```diff
@@ -249,13 +249,13 @@ def __init__(
 
     def apply(  # type: ignore
         self, document: Document, pdf_path: Optional[str] = None, **kwargs: Any
-    ) -> Iterator[Sentence]:
+    ) -> Document:
         # The document is the Document model
         text = document.text
 
         # Only return sentences, if no exceptions occur during parsing
```

Comment:
Let's update this comment, too. Since we are returning a document now, not sentences.

Reply:
Good catch. I update it by simply removing the comment and adding a docstring.